src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineGroupConcatFieldDescriptor.php

```diff
@@ -61,9 +71,15 @@ public function getSelect()
     }
 
     /**
-     * Returns all the joins required for this field.
-     *
-     * @return DoctrineJoinDescriptor[]
+     * {@inheritdoc}
+     */
+    public function getSimpleSelect()
```

Comment:
that seems not to be a problem. i can filter for accounts but it is not displayed - no error.